libs/sdk-backend-tiger/src/convertors/toBackend/afm/TotalsConverter.ts

```diff
@@ -41,6 +51,125 @@ export function convertTotals(def: IExecutionDefinition): Total[] {
             return tigerTotal;
         });
     }).filter((total): total is Total => total !== undefined);
+
+    /**
+     * With new column totals, we need to have in consideration the cases where same totals/subtotals are selected for rows and columns.
+     * This new cases would require extra configuration so Tiger AFM is able to compute properly the result.
+     * These new cases are -
+     *   1. Grand totals - is the value obtained from row and column totals.
+     *   2. Marginal totals - is the value obtained within the subgroups from row and column subtotals.
+     *
+     * For `Grand Totals`, in Tiger AFM, you specify that you want total of totals with total that have only "measureGroup" present.
+     * For `Marginal Total`, in Tiger AFM, would need to iterate through both dimensions and obtain the missing totalDimensions items
+     * based on the attribute and column identifiers order in buckets.
+     */
+
+    const attributeBucket = bucketsFind(buckets, "attribute");
```

Comment:
I think we have constants for these "attribute", "columns", please use them.

Reply:
They are from an external sdk package that cannot be reached from `sdk-backend-tiger`.
But have added them as constants.